packages/editor/src/components/warning/index.js

```diff
@@ -40,13 +40,13 @@ function Warning( { className, actions, children, secondaryActions } ) {
 						/>
 					) }
 					renderContent={ () => (
-						<MenuGroup label={ __( 'More options' ) }>
+						<NavigableMenu orientation="vertical" className="editor-warning-menu__content">
```

Comment:
This seems like a lower-level component and API-wise it seems better to have a `MenuGroup` wrapping `MenuItem`s which makes me wonder if we can't just support no passing a label to `MenuGroup` instead of this?

Reply:
@youknowriad Thanks for reviewing. Yes, you are right. better to use `MenuGroup` without label as `NavigableMenu` is lower-level component. I will make this change soon.